optuna/_convert_positional_args.py

```diff
@@ -80,7 +80,7 @@ def converter_wrapper(*args: Any, **kwargs: Any) -> "_T":
 
             kwargs.update(inferred_kwargs)
 
-            return func(**kwargs)
+            return func(**kwargs)  # type: ignore
```

Comment:
```suggestion
            return func(**kwargs)  # type: ignore[call-arg]
```

Reply:
I'm not sure, but it looks similar to this bug report https://github.com/python/mypy/issues/17960, so I think it's OK to ignore it for now.

Reply:
Thank you for your suggestion.